magnum_cluster_api/cmd/image_loader.py

```diff
@@ -57,6 +58,10 @@
     required=True,
     help="Target image repository",
 )
+@click.option(
+    "--manifest",
```

Comment:
`--images-manifest` would make it more explicit this is about images and not some other type of manifest (now or in the future). wdyt?

Reply:
@gtirloni we have updated this now to be `--images-manifest`